### ISSUE TITLE
Mistake in code

### ISSUE THREAD

**Issue by prashastbindal** [Abhipanda4/Feature-Generating-Networks#3]
There is a big mistake in the code. The classifiers are trained on a concatenation of image features and the class attributes. Obviously, it should be just the image features.

**Comment by Pamulapati13** [comment]
Yes I too had the same issue, why is he concatenating visual and semantic features.

Did you try training it with out concatenation?
